src/stactools/nrcan_spot_ortho/geobase_ftp.py

```diff
@@ -1,7 +1,7 @@
 import os
 from ftplib import FTP
 from time import sleep
-from tests.test_utils import hrefs, href_thumb
+from stactools.nrcan_spot_ortho.constants_test import hrefs, href_thumb
```

Comment:
So why do we need "test" constants in the package? If they're "test" they shouldn't be distributed w/ the package, but if they're required for proper operation of the package then we should rename them to something not "test", IMO.

Reply:
Good point.

Given the current structure of the package and that we want to avoid contacting the FTP during testing, I can't think of an elegant way to perform tests without some test-specific code in the public package. I've tried to minimise that with the latest commit by writing test item content to some files and conditioning the public package to recognise that it should avoid contacting the FTP if provided with those files. Maybe not ideal.

I can open an issue to refactor the package if you think it's worth doing.

Reply:
> I can open an issue to restructure the package if you think it's worth doing.

Yeah, hopefully it wouldn't be too bad to just build a configurable FTP class (with sensible defaults) and then do the custom testing configuration in the `tests` module.